src/main/java/com/powsybl/openloadflow/util/ParameterConstants.java

```diff
@@ -26,6 +26,9 @@
     public static final String VOLTAGE_REMOTE_CONTROL_PARAM_NAME = "voltageRemoteControl";
     public static final boolean VOLTAGE_REMOTE_CONTROL_DEFAULT_VALUE = true;
 
+    public static final String REACTIVE_POWER_REMOTE_CONTROL_PARAM_NAME = "reactivePowerRemoteControl";
+    public static final boolean REACTIVE_POWER_REMOTE_CONTROL_DEFAULT_VALUE = true;
```

Comment:
This sould be `false` bu default.

Reply:
This will disable reactive power remote control by default. This mean that setting a RegulationMode to ReactivePower on a generator will have no effect unless we also enable this parameter explictly.
Are we sure we want this behavior ? If yes I would also add a warning somewhere that warn the user its generator configuration is being overriden by a global parameter default value.

Reply:
Yes, I am sure because sometimes it could lead to unfeasible equation system. But, now I wonder about the API on powsybl-core. If a generator has a remote reactive target and an active regulation, if we ignore it, it means that we have to found a `targetQ` that matches locally. @MioRtia this is our issue :-) We have to store from CGMES file both SSH local targetQ and the remote targetQ... what do you think?